package.json

```diff
@@ -38,12 +40,25 @@
     "eslint-config-airbnb-base": "~14.2.1"
   },
   "devDependencies": {
```

Comment:
Since we now have a significant number of dependencies that won't be needed for non-Angular projects, what do you think of updating the "Installation" section of README.md to provide more explicit guidance about using `npm view` vs `npx install-peerdeps`?

I'm thinking something like a note in the `install-peerdeps` section saying "For non-Angular projects, this will install a significant number of unneeded dependencies"

Reply:
@rajsite will verify the behavior of the future version of `npm`. In the spirit of progress, I will move this discussion to an issue.